frontend/src/services/PlanService.ts

```diff
@@ -150,3 +152,93 @@ export const setPrimaryPlan = (userId: number, planId: number) =>
       "Content-Type": "application/json",
     },
   }).then(response => response.json());
+
+/**
+ * Gets changelogs for a plan
+ * @param planId The id of the plan
+ * @param studentId The id of the student
+ */
+export const fetchChangelogs = (planId: number, studentId: number) =>
+  fetch(`/api/users/${studentId}/plans/${planId}/plan_changelogs`, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: "Token " + getAuthToken(),
+    },
+  }).then(response =>
+    response.json().then((changeLogs: IChangeLog[]) => {
+      return changeLogs.map((changeLog: IChangeLog) => ({
+        ...changeLog,
+        createdAt: new Date(changeLog.createdAt), // convert string timestamp to a Date object
+        updatedAt: new Date(changeLog.updatedAt), // convert string timestamp to a Date object
+      }));
+    })
+  );
+
+/**
+ * Sends a comment for a plan
+ */
+export const sendChangeLog = (
+  planId: number,
+  studentId: number,
+  authorId: number,
+  log: string
+): Promise<IChangeLog> =>
+  fetch(`/api/users/${studentId}/plans/${planId}/plan_changelogs/`, {
+    method: "POST",
+    body: JSON.stringify({ log: log, author_id: authorId }),
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: "Token " + getAuthToken(),
+    },
+  }).then(response => response.json());
+
+/**
+ * Gets comments for a plan
+ * @param planId The id of the plan
+ * @param studentId The id of the student
+ */
+export const fetchComments = (planId: number, studentId: number) =>
```

Comment:
I think comments belong in the plan service as students will also be viewing/adding comments